pkg/proc/bininfo.go

```diff
@@ -52,6 +52,9 @@ type BinaryInfo struct {
 
 	debugInfoDirectories []string
 
+	// BuilID of this binary.
```

Comment:
```suggestion
	// BuildID of this binary.
```

😄 sorry for the drive-bys!

Reply:
Thanks :)!